mllm/models/qwen_ascend/qwen_ascend_graph_ops.hpp

```
@@ -0,0 +1,207 @@
+}
+
+inline bool isQwenAscendDecoderGraphEnabled() {
+  const char* env = std::getenv("MLLM_ASCEND_QWEN_DECODER_GRAPH");
+  return env == nullptr || env[0] != '0';
+}
+
+inline int32_t getQwenAscendDecoderGraphSetupBucketSize() {
+  const char* env = std::getenv("MLLM_ASCEND_QWEN_DECODER_GRAPH_SETUP_BUCKET");
+  if (env == nullptr || env[0] == '\0') return 0;
+  char* end = nullptr;
+  const long parsed = std::strtol(env, &end, 10);
```

Clang (14.0.6)

[error] consider replacing 'long' with 'int64'

(google-runtime-int,-warnings-as-errors)